.circleci/config.yml

```diff
@@ -4,43 +4,32 @@ orbs:
   python: circleci/python@0.3.0
 
 jobs:
-  terratest:
+  build-and-release:
     docker:
-      - image: trussworks/circleci-docker-primary:c542b22c7fb95db0a1bbe043928a457ae6fbeaca
-        environment:
-          - TEST_RESULTS: /tmp/test-results
+      - image: trussworks/circleci-docker-primary:ghr
```

Comment:
Since you're killing this you can probs delete `terraform_aws_iam_sleuth_test.go` too.

Reply:
oh shoot. you're right.

Reply:
i think i deleted it.